.github/workflows/osx.yml

```diff
@@ -0,0 +1,32 @@
+name: OSX-build
+
+on: [push]
+
+jobs:
+  osx-clang-omp:
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        - {shared: "ON", build_type: "Debug", name: "omp/debug/shared"}
+        - {shared: "OFF", build_type: "Release", name: "omp/release/static"}
+    name: ${{ matrix.config.name }}
+    runs-on: [macos-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: brew install libomp
+    - name: info
+      run: |
+        g++ -v
```

Comment:
Will `g++` actually be used for the compilation? I thought we are testing `AppleClang`.

Reply:
The default g++ on mac is actually AppleClang, which creates problems to a lot of unknowing mac users at first (thinking they have are using g++, but they are not by default).